pages/ksphere/kommander/latest/operations/cloud-providers/azure/index.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: layout.pug
+navigationTitle: Azure Cloud Provider
+title: Azure Cloud Provider
+excerpt: Managing the Azure cloud provider used by Kommander
+---
+
+Kommander needs authentication keys to Azure to automate its provisioning. It is possible to have many accounts for a single cloud provider.
+
+### Configuring an Azure cloud provider
+
+When creating an Azure cloud provider, you need a service principal with a set of minimum  capabilities. This requires you to have `Contributor` and `User Access Administrator` rights on Azure.
+
+#### Create a service principal via CLI commands
+
+Find the `SUBSCRIPTION_ID` of your account, named `id` in the output of this command:
+```
+az account show
+```
+
+Create the service principal for the provider. This returns all the data needed for the secret we will create later:
```

Comment:
s/Create the service principal for the provider. This returns all the data needed for the secret we will create later:/Create the service principal./

Reply:
How about we add the output of this as an example.